docs/source/community/persons_of_interest.rst

```diff
@@ -190,23 +225,20 @@ Build + CI
 -  Nikita Shulga (`malfet <https://github.com/malfet>`__)
 -  Eli Uriegas (`seemethere <https://github.com/seemethere>`__)
 -  Alban Desmaison (`alband <https://github.com/alband>`__)
--  Mikey Dagitses (`dagitses <https://github.com/dagitses>`__)
--  Omkar Salpekar (`osalpekar <https://github.com/osalpekar>`__)
--  Zain Rizvi (`ZainRizvi <https://github.com/ZainRizvi>`__)
--  Nirav Mehta (`mehtanirav <https://github.com/mehtanirav>`__)
 -  Andrey Talman (`atalman <https://github.com/atalman>`__)
+-  Zain Rizvi (`ZainRizvi <https://github.com/ZainRizvi>`__)
+-  (emeritys) Mikey Dagitses (`dagitses <https://github.com/dagitses>`__)
+-  (emeritys) Omkar Salpekar (`osalpekar <https://github.com/osalpekar>`__)
+-  (emeritys) Nirav Mehta (`mehtanirav <https://github.com/mehtanirav>`__)
 -  (emeritus) Zhuojie Zhou (`zhouzhuojie <https://github.com/zhouzhuojie>`__)
 -  (emeritus) Edward Yang (`ezyang <https://github.com/ezyang>`__)
 -  (emeritus) Karl Ostmo (`kostmo <https://github.com/kostmo>`__)
 
 Performance Tools
 ~~~~~~~~~~~~~~~~~
 
--  Adnan Aziz (`adnanaziz <https://github.com/adnanaziz>`__)
--  CK Luk (`ckluk <https://github.com/ckluk>`__)
 -  Taylor Robie (`robieta <https://github.com/robieta>`__)
 -  Xu Zhao (`xuzhao9 <https://github.com/xuzhao9>`__)
--  Geeta Chauhan (`chauhang <https://github.com/chauhang>`__)
```

Comment:
these folks didn't get put in emeritus?

Reply:
Yes people that were listed but didn't contribute code or comments to the category were removed instead of emeritus.
In this case, the only comment is about asking the user to use the latest version. Which I'm not sure is relevant as a maintainer?